physionet-django/user/models.py

```diff
@@ -701,6 +702,26 @@ def is_latest_by_user(self):
         else:
             return False
 
+    def apply_decision(self, decision, responder):
+        """
+        Reject (1), accept (2), or withdraw (3) a credentialing application.
+
+        Args:
+            decision (int): 1 is reject, 2 is accept, 3 is withdraw.
+            responder (str): User object
+        """
+        # if accepted, credential the user
```

Comment:
Shouldn't this be atomic? In the case that he has been approved, we need both to happen at the same time.

Reply:
yep, good call!